src/schema/pathwayTimeseries.v1.json

```
@@ -0,0 +1,309 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://pathways-library.rmi.org/schema/pathwayTimeseries.v1.json",
+  "title": "PBTAR Scenarios Timeseries Schema",
+  "description": "A schema for the scenarios metadata dataset in PBTAR.",
+  "type": "object",
+  "$defs": {
+    "sectorDisplayName": {
+      "type": "string",
+      "enum": [
+        "Land Use",
+        "Agriculture",
+        "Buildings",
+        "Industry",
+        "Steel",
+        "Cement",
+        "Chemicals",
+        "Coal Mining",
+        "Oil (Upstream)",
+        "Gas (Upstream)",
+        "Power",
+        "Automotive",
+        "Transport",
+        "Road transport",
+        "Aviation",
+        "Rail",
+        "Shipping",
+        "Other"
+      ]
+    }
+  },
+  "properties": {
+    "$schema": {
+      "description": "URI of the schema that validates this document (see https://json-schema.org/).",
+      "type": "string",
+      "maxLength": 1000
+    },
+    "id": {
+      "description": "The unique identifier for a scenario.",
+      "type": "string",
+      "maxLength": 100
+    },
+    "name": {
+      "description": "Name of the scenario.",
+      "type": "string",
+      "maxLength": 100
+    },
+    "description": {
+      "description": "Brief description of the scenario.",
+      "type": "string",
+      "pattern": "\\.$",
+      "maxLength": 100
+    },
+    "publisher": {
+      "description": "Publisher of the scenario.",
+      "type": "string",
+      "maxLength": 100
+    },
+    "publicationName": {
+      "description": "Publisher of the scenario.",
+      "type": "string",
+      "maxLength": 100
+    },
+    "publicationYear": {
+      "description": "Year that the scenario was published.",
+      "type": "integer",
+      "minimum": 1900,
+      "maximum": 2030
+    },
+    "pathwayName": {
+      "description": "Name of the assosciated scenario.",
+      "type": "string",
+      "maxLength": 100
+    },
+    "source": {
+      "description": "Data source for the scenario data.",
+      "type": "string",
+      "maxLength": 500
+    },
+    "emissionsScope": {
+      "description": "Defines which greenhouse gases are covered in the scenario’s emissions accounting.",
```

```suggestion
      "description": "Defines which greenhouse gases are covered in the pathway's modeled emissions.",
```
I would choose a formulation that is closer to what is normally found in pathway publications.